### ISSUE TITLE
Ansible request for RHEL/CentOS 8 libmpc-devel & libdwarf-devel

### ISSUE THREAD

**Issue by Willsparker** [AdoptOpenJDK/openjdk-infrastructure#1750]
Ref: https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1743

- [x] Missing install

Details:
The `libmpc-devel` and `libdwarf-devel` packages have moved to the CodeReady Linux Builder (CRB) repo, and the PowerTools repo for RHEL and CentOS respectively (though we may be able to use PowerTools for RHEL..?).
The referred PR above will remove them from the list of packages to install in the meantime, however we need both the packages to build OpenJ9, so we need to find a way of getting them in.

**Comment by Willsparker** [comment]
Checking [this](https://centos.pkgs.org/8/centos-powertools-x86_64/) we can get `libmpc-devel` and `libdwarf-devel` from the repos,so I'll test with CentOS and then try find a RHEL machine to test on, somewhere

**Comment by Willsparker** [comment]
Checking on a CentOS8 Vagrant VM, the Powertools repo is already installed from the start, it's just a case of enabling it. Unfortunately, attempting to enable it either through `yum` or `dnf` commands didn't seem to work, but enabling it manually did (i.e. using `vi`). This would be easy enough to ansiblise, however.

**Comment by Willsparker** [comment]
Having checked the `test-aws-rhel8-x64-1` machine, unfortunately, the PowerTools repository doesn't exist on the machine ( and in a brief Google search, I couldn't find if it to be installable ). That unfortunately leaves us with having to deal with the CRB repo and `subscription-manager` if we want to build OpenJ9 on RHEL8+. Running a few commands on the machine whilst I was testing, I found that that particular machine hasn't been registered yet, and so repos were available via `subscription-manager`. 

```
[root@54 ~]# subscription-manager repos
This system has no repositories available through subscriptions.
[root@54 ~]# subscription-manager list --available
This system is not yet registered. Try 'subscription-manager register --help' for more information.
```

While we can likely ansibilise the `subscription-manager` commands, registering the subscription may not be possible.

**Comment by Willsparker** [comment]
I've had a quick look trying to install it manually, via `wget`ing from [here](https://centos.pkgs.org/8/centos-powertools-x86_64/libdwarf-devel-20180129-4.el8.x86_64.rpm.html) , however, it ends up leading down a dependency path. `libdwarf-devel` depends on `libdwarf`, which one CAN install on `RHEL8` (via the AppStream repository), but `rpm` still seems to think there are unmet dependencies (possibly due to trying to install a `CentOS8 libdwarf-devel`, not a `RHEL8` one?)

Either way, it's not the 'proper' way to do this.

**Comment by Willsparker** [comment]
@gdams / @sxa  Can either of you see any way of registering a RHEL8 subscription via Ansible? I'm not even sure if we can do it manually, given our machines are provided to us ...

**Comment by gdams** [comment]
I don't think that we have a RH subscription so it's unlikely that we can do that, unless @sxa can pull some strings now that he's on the inside??

**Comment by Willsparker** [comment]
ref: #2008 
Lucky this came about - this should help in getting the packages we need for RHEL8 :-)

**Comment by sxa** [comment]
We have an outstanding request for a ROSI subscription which will allow us to run RHEL but it has not come through yet.

**Comment by sxa** [comment]
> We have an outstanding request for a ROSI subscription which will allow us to run RHEL but it has not come through yet.

ROSI subscription has now apparently been approved so we should be able to progress this one ...

**Comment by sxa** [comment]
Bumping this to October as I'll hopefully have some time next month to look at setting this up
